addons/sourcemod/translations/freak_fortress_2.phrases.txt

```diff
@@ -277,6 +277,11 @@
 		"#format" "{1:s}"
 		"en"	"FF2 monologues: {1}."
 	}
+	"ff2_companion"
+	{
+		"#format" "{1:s}"
+		"en"	"FF2 Companions: {1}"
```

Comment:
What about you change this message to something like "You won't be selected as companion to a boss" and "You will be selected as companion to a boss"?